### ISSUE TITLE
which qpkg for ts-431p 

### ISSUE THREAD

**Issue by dnerpzza** [tailscale/tailscale-qpkg#119]
Hey guys my Nas is apparently on arm 32bit
I can't find any qpkg based on arm 32 it's only 32 and 64 and so on.
can someone help me ?
Which qpkg I have to use?

Thanks in advance.

**Comment by athinaok** [comment]
Please use Tailscale_1.48.2-1_arm-x41.qpkg

**Comment by dnerpzza** [comment]
> Please use Tailscale_1.48.2-1_arm-x41.qpkg

I did that now. But instead of cookie not present Error i get this:

**Internal Server Error**
The server encountered an internal error or misconfiguration and was unable to complete your request.

Please contact the server administrator at admin@NAS to inform them of the time this error occurred, and the actions you performed just before this error.

More information about this error may be available in the server error log.



Do you know, why am i getting this? What can i do to fix it?
Appreciate the help.

**Comment by DentonGentry** [comment]
You see the internal server error from where? From the Tailscale UI, which you get to by clicking on the Tailscale icon in the App Center?

Are you able to ssh to the QNAP device? From a shell, can you: `ps auxw | grep -i tailscale`
to check whether the daemon is running.

**Comment by dnerpzza** [comment]
> You see the internal server error from where? From the Tailscale UI, which you get to by clicking on the Tailscale icon in the App Center?
> 
Yes exactly after clicking on the UI on the app center or rather just from the "home screen". same thing.



> Are you able to ssh to the QNAP device? From a shell, can you: `ps auxw | grep -i tailscale` to check whether the daemon is running.

the thing with the ssh i dont understand. Can you explain detailed what exactly i have to do ?

**Comment by dnerpzza** [comment]
> ps auxw | grep -i tailscale

if i did it right it should be 

[~] # ps auxw | grep -i tailscale
 2302 admin     22156 S   /share/CACHEDEV2_DATA/.qpkg/Tailscale/tailscaled --port XXXXX --statedir=/share/CACHEDEV2_DATA/.qpkg/Tailscale/state --socket=/tmp/tailscale/tailscaled.sock
30113 admin     YYYYYY S   tailscaled --socket=/tmp/tailscaled.sock --state=mem: --statedir=/tmp --tun=userspace-networking
30122 admin      ZZZ S   tailscale --socket=/tmp/tailscaled.sock up --accept-dns=false
30445 admin       TTT S   grep -i tailscale
[~] #

The numbers i changed with XYZT because i dont know if im supposed to share them.

Is that allright like that?

**Comment by DentonGentry** [comment]
There somehow are two versions of Tailscale running. The first appears to be the QNAP package installed through the App Center, the second is running in `userspace-networking` mode which the QNAP package does not use.

1. `/share/CACHEDEV2_DATA/.qpkg/Tailscale/tailscaled --port XXXXX --statedir=/share/CACHEDEV2_DATA/.qpkg/Tailscale/state --socket=/tmp/tailscale/tailscaled.sock`
1. `tailscaled --socket=/tmp/tailscaled.sock --state=mem: --statedir=/tmp --tun=userspace-networking`

I would suggest:
1. uninstall the package in the App Center
1. Reboot the QNAP
1. install the package in the App Center

**Comment by dnerpzza** [comment]
> ps auxw | grep -i tailscale

I did that --> the userspace is no more but still the same error with the internal server.

ps auxw | grep -i tailscale shows now:
20436 admin     28212 S   /share/CACHEDEV2_DATA/.qpkg/Tailscale/tailscaled --port XXXXX --statedir=/share/CACHEDEV2_DATA/.qpkg/Tailscale/state --socket=/tmp/tailscale/tailscaled.sock
25795 admin       452 R   grep -i tailscale
[~] #

**Comment by dnerpzza** [comment]
can someone help?

**Comment by daxliniere** [comment]
I got an error when I first installed and found that I needed to start the service (in App Center) first.
